content/blog/2023/09/06/2023-09-06-artifactory-bandwidth-reduction.adoc

```diff
@@ -0,0 +1,38 @@
+---
+layout: post
+title: "Artifactory Bandwidth Reduction"
+tags:
+- infrastructure
+- artifactory
+- jfrog
+authors:
+- markewaite
+- dduportal
+opengraph:
+  image: /images/post-images/2023/09/06/2023-09-06-artifactory-bandwidth-reduction.png
+discourse: true
+---
+
+image:/images/post-images/2023/09/06/2023-09-06-artifactory-bandwidth-reduction.png[image,width=839]
+
+link:https://jfrog.com/[JFrog] has been a sponsor of the Jenkins project for many years.
+We're delighted that they continue to sponsor the Jenkins project and continue to provide our artifact hosting service, link:https://repo.jenkins-ci.org[repo.jenkins-ci.org].
+
+Releases, incremental development builds, and snapshots of Jenkins core, Jenkins tooling, Jenkins plugins, and Jenkins infrastructure components are hosted on JFrog Artifactory.
+The worldwide Jenkins community has been well served for many years by JFrog's hosted artifact repository.
+
+// == Bandwidth misuse
+
+// We've had instances in the past when we detected excessive or unhelpful use of the Jenkins artifact repository.
+// During peak usage, over 50 TB of data was transferred each month.
+
+// JFrog asked us to find ways to reduce that usage.
+// We've found a low impact way to reduce misuse of the artifact repository bandwidth while continuing to meet the needs of the Jenkins project.
+
+== Maven Central
+
+Effective Thursday September 7, 2023, we'll stop caching the Apache Maven Central repository.
+The change will be transparent to Jenkins developers because Apache Maven automatically appends the Apache Maven Central repository to its list of repositories.
```

Comment:
```suggestion
The change will be transparent to Jenkins developers because Apache Maven automatically appends the Maven Central repository to its list of repositories.
```